### ISSUE TITLE
After expiration of a day, authorization from the database ceases to work

### ISSUE THREAD

**Issue by kalistrat** [moquette-io/moquette#360]
After expiration of a day, authorization from the database ceases to work.
when you try to connect, an error occurs:
```
Bad username or password (4)
	at org.eclipse.paho.client.mqttv3.internal.ExceptionHelper.createMqttException(ExceptionHelper.java:28)
	at org.eclipse.paho.client.mqttv3.internal.ClientState.notifyReceivedAck(ClientState.java:885)
	at org.eclipse.paho.client.mqttv3.internal.CommsReceiver.run(CommsReceiver.java:118)
	at java.lang.Thread.run(Thread.java:745)
```
A simple reboot of the broker without changing the settings helps. Authorization begins to work, but not for long. After a day, the error is reproduced again.

OS version: Ubuntu 17.04 (GNU/Linux 4.10.0-22-generic x86_64)
Broker version: 0.9
MySQL version: 5.7.19-0ubuntu0.17.04.1

**Comment by kalistrat** [comment]
Exactly the same problem occurs with the mosquitto: 
https://github.com/jpmens/mosquitto-auth-plug/issues/238

**Comment by andsel** [comment]
It's due to the long lasting connections to DB which must be managed by a connection pool.
Fixed with ba2b8b44d3a93ab4498480e33016605791500406
